docs/source/contributing.rst

```diff
@@ -0,0 +1,43 @@
+Contributing to the template
+===============================================================================
+
+Find (or make) a new GitHub issue
+-------------------------------------------------------------------------------
+
+Add yourself as the assignee on an existing issue so that we know who's working on what. ( If you're not actively working on an issue, unassign yourself 😉 )
```

Comment:
I think we should include a statement like, "If you don't have permissions to assign yourself to an issue, please reach out to us on #lincc-dev (maybe?) or lincc-frameworks-team@lists.lsst.org".

Reply:
If we refer to #lincc-dev we'd have to say "in the LSSTC Slack workspace", which not everybody who sees this page will have access to.  I'm not sure if @jeremykubica configured the team list to be reachable by people who aren't subscribed to it (is it meant for team discussion or for people outside the team to reach the entire team?).  

I think we may want to think carefully about how we encourage people not on the team to reach out - this applies, as well, for people with questions on the website or with questions about any of our GitHub repos etc.  Do we already have a consistent statement on this elsewhere that we can use here?

Reply:
For now, we can recommend folks create a new issue to start a conversation.